web/src/proto/step128_2020.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package step189_2020;
+
+message PushInfo {
+  string push_handle = 1;
+
+  message StateInfo {
+    int32 state = 1;
+    string stage = 2;
+    int64 attempt = 7;
```

Comment:
It might be a bit late, but it seems we don't have to keep attempt as 7, are we still planning to change it or keep it?

Reply:
Good point. I'll fix this before merging.